assets/css/main.scss

```diff
@@ -56,6 +56,77 @@
   vertical-align: middle;
 }
 
+/* Styles the social media share buttons on the post pages
+   From: https://nickjanetakis.com/blog/add-social-share-buttons-to-your-static-site-without-tracking */
+.social-share-floating-bar {
+  position: fixed;
+  top: 50%;
+  -webkit-transform: translateY(-50%);
+  transform: translateY(-50%);
+}
+
+.social-share-floating-bar a {
+  display: block;
+  transition: all 0.3s ease;
+  padding: 16px;
+  color: #fff;
+  font-size: 20px;
+  text-align: center;
+}
+
+.social-share-btns-container {
+  font-size: 14px;
+  text-align: left;
+}
+
+.social-share-btns {
+  display: inline-block;
+}
+
+.social-share-btns .share-btn {
+  margin: 0 5px;
+  padding: 4px 8px;
+  color: #fff;
+}
+
+.social-share-btns .share-btn i {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+.share-btn {
+  background-color: #95a5a6;
+}
+
+.share-btn:hover, .share-btn:active, .share-btn:visited {
+  background-color: #798d8f;
+  text-decoration: none;
+}
+
+.share-btn-twitter {
+  background-color: #0087ba;
+}
+
+.share-btn-twitter:hover, .share-btn-twitter:active, .share-btn-twitter:visited {
```

Comment:
this is a lot cleaner if you nest the styles - this is scss

Reply:
Thanks @johnomalley! How does the latest version look?

Reply:
And I just nested and cleaned it up even further.

Reply:
looks great thanks Jay